src/ert/config/ert_config.py

```diff
@@ -59,7 +58,7 @@
 from .workflow_job import ErtScriptLoadFailure, WorkflowJob
 
 if TYPE_CHECKING:
-    from importlib.abc import FileLoader
+    pass
```

Comment:
Instead of passing `TYPE_CHECKING`, can we just remove it?

Reply:
ay yes, should have. Sorry I clicked the merge button too soon.